.env.example

```
@@ -0,0 +1,58 @@
+
+# Optional: Monitoring Configuration
+# PROMETHEUS_PORT=9090
+# GRAFANA_PORT=3000
+
+# Development Configuration
+NODE_ENV=development
+PORT=3002
+
+# External Services Configuration (Development)
+VAULT_ENABLED=false
+REDIS_ENABLED=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The REDIS_ENABLED key should go before the VAULT_ENABLED key

(UnorderedKey)